app/jobs/spotlight_search/export_job.rb

```diff
@@ -0,0 +1,33 @@
+module SpotlightSearch
+  class ExportJob < ApplicationJob
+    def perform(email, records, columns = nil, filters = nil)
+      # data = get_data(records.class, filters)
+      file_path = create_excel(records.as_json, records.class.name)
```

Comment:
@mikevic Why are we going with excel?

Reply:
A csv of 75 MB is only 12 MB in excel. Excel is compressed format. We can go back to csv if required. I was thinking of the file size only